examples/workshop-fomo-generalized/index.rsh

```diff
@@ -0,0 +1,68 @@
+'reach 0.1';
+
+// FOMO Workshop generalized to last N winners
+const NUM_OF_WINNERS = 3;
```

Comment:
I think your workshop text should comment on how Reach doesn't support arbitrary sized arrays, but we CAN write a program that is generic in the size of the array and then specialize it when we compile.